cms/jinja2/templates/components/streamfield/dataset_link_block.html

```diff
@@ -0,0 +1,4 @@
+<li class="ons-list__item">
+    <p><a href="{{ value.website_url or value.url }}">{{ value.title }}</a></p>
```

Comment:
```suggestion
    <p><a href="{{ value.website_url|default:value.url }}">{{ value.title }}</a></p>
```

Reply:
Addressed in https://github.com/ONSdigital/dis-wagtail/pull/123/commits/6ce4cf0620a7652c93c212421964b07cc5b5d86f

Reply:
Reverting this as it was causing rendering issues
```
  File "/app/cms/jinja2/templates/components/streamfield/dataset_link_block.html", line 2, in template
    <p><a href="{{ value.website_url|default:value.url }}">{{ value.title }}</a></p>
    ^^^^^^^^^^^^^^^^^^^^^^^^^
django.template.exceptions.TemplateSyntaxError: ("expected token 'end of print statement', got ':'",)
```

Is there some other change required to make your suggestion work as intended?

Reply:
The syntax Jake suggested is for Django templates. Since we are using Jinja for the frontend templates, you will need to update it to `{{ value.website_url|default(value.url) }}`